stable/rabbitmq/values-production.yaml

```diff
@@ -96,12 +96,13 @@ service:
   ## Node port
   ## ref: https://github.com/bitnami/bitnami-docker-rabbitmq#environment-variables
   ##
-  nodePort: 5672
+  nodePort: 30672
```

Comment:
We should not be setting a default value to `nodePort`. As the README.md says, the default value should be random:

> random available between 30000-32767

Reply:
As you can check in the `values.yaml`, this parameter is commented.

Reply:
@juan131 that makes sense. The service is of type ``ClusterIP`` in the production values anyway. I've already made the necessary changes.